docs/userguide/calling.rst

```diff
@@ -454,7 +454,7 @@ Each option has its advantages and disadvantages.
 
 json -- JSON is supported in many programming languages, is now
     a standard part of Python (since 2.6), and is fairly fast to decode
-    using the modern Python libraries, such as :pypi:`simplejson`.
```

Comment:
and is fairly fast to decode
    using the modern Python libraries, such as :pypi:`simplejson`.
    using modern Python libraries, such as :pypi:`simplejson`.

this part can be completely removed or rephrased as celery do not use simplejson anymore

Reply:
Done in 7219ca3 :+1: